src/examples/source/GUI.cpp

```diff
@@ -316,22 +316,37 @@ int main(int argc, char **argv)
         for (const std::string& pathname : {application_handler_.mzML_dir_, application_handler_.features_in_dir_, application_handler_.features_out_dir_}) {
           fs::create_directories(fs::path(pathname));
         }
-        for (ApplicationHandler::Command& cmd : application_handler_.commands_)
-        {
-          for (std::pair<const std::string, Filenames>& p : cmd.dynamic_filenames)
+        std::vector<ApplicationHandler::Command> commands;
+        bool commands_created = true;
+        CreateCommand createCommand(application_handler_);
+        for (const auto& command_name : application_handler_.sequenceHandler_.getWorkflow()) {
```

Comment:
Nice.  I am thinking we made a good decision in moving towards a string representation of the workflow steps that are converted on the fly as shown here.  I think this will make "headless" mode much easier to implement.

On that note, do you think it would make sense to refactor this chunk of code into its own method?